cookbook/specs/solana.json

```diff
@@ -994,6 +1318,87 @@
                                     "encoding": "base64"
                                 },
                                 "api_name": "getBlock"
+                            },
+                            {
+                                "function_tag": "SUBSCRIBE",
+                                "api_name": "accountSubscribe"
+                            },
+                            {
+                                "function_template": "{\"jsonrpc\":\"2.0\",\"method\":\"accountUnsubscribe\",\"params\":[0],\"id\":1}",
```

Comment:
Why is it constant 0? I think it should be `%s`.
Please test this, and the rest of the unsubscribe methods as well

Reply:
After testing, it should be %d. see here: https://github.com/lavanet/lava/pull/1808/commits/b30d822b3a4eee1800246a0bd41272274215d600